app/controllers/tenants/projects_controller.rb

```diff
@@ -1,7 +1,10 @@
 class Tenants::ProjectsController < ApplicationController
 
   def index
-    @tenant = Tenant.find_by(slug: params[:slug])
+    @tenant = Tenant.where(slug: params[:slug])
+                    .where(active: true)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.